### PR TITLE
mkcloud driver improvements

### PR DESCRIPTION
**Pull request by toabctl**

### CODE REVIEW

**Comment by toabctl** [comment]
Not tested...

**Comment by bmwiedemann** [comment]
the first commit needs to also rename the 2nd driver: scripts/lib/mkcloud-dirmaint.sh

File: scripts/mkcloud [hunk 1]
```diff
@@ -376,11 +376,6 @@ function onhost_add_etchosts_entries
     grep -q crowbar /etc/hosts || echo "$adminip crowbar.$cloudfqdn crowbar" >> /etc/hosts
 }
 
-function onhost_enable_ksm
-{
-    ${mkclouddriver}_onhost_enable_ksm
```
Comment (bmwiedemann) [inline scripts/mkcloud]: actually there is a `libvirt_enable_ksm` so when you drop the `onhost_` it could work

**Comment by toabctl** [comment]
> the first commit needs to also rename the 2nd driver: scripts/lib/mkcloud-dirmaint.sh

Done

File: scripts/mkcloud (hunk 1 above)
Comment (toabctl) [reply scripts/mkcloud]: @bmwiedemann the code is executed in the ```libvirt_do_prepare``` step. I added that to the commit msg. Is that enough?

**Pull request merged**